week-3/reading-list/script.js

```diff
@@ -1,3 +1,23 @@
+function readingList(books) {
+  const readingList = document.getElementById("reading-list");
+
+  for (const item of books) {
+    const list = document.createElement("li");
+
+    list.setAttribute("class", item.alreadyRead ? "teal" : "red");
```

Comment:
I'm going to be _really_ picky, but hopefully this is a useful note!

As much as possible, it's good to keep your styling separate from logic. You could use class names like "read" and "unread", then only use CSS to decide what the colour should be.

This is really useful for modifying your site in the future, as all colour choices are together in the CSS. There is also a scenario where your site might have different themes, for example, a light and dark mode. The colours for read and unread books might not always be the same in each mode.

Reply:
That makes sense, I will keep that in mind, Thank you